custom/opm/beneficiary.py

```diff
@@ -508,7 +508,7 @@ def _test(form):
 
     @property
     def child_received_measles_vaccine(self):
-        if self.child_age == 12:
+        if self.child_age == 12 and self.block == 'atri':
```

Comment:
These three properties (`child_received_measles_vaccine`, `child_birth_registered`, and `child_weighed_once`) do not specify a block, but they are referred via the property `child_condition_four`, which DOES check for the appropriate block.  I've moved that check here as well to make sure it always happens.